### PR TITLE
Claimant Representative Automation and Refactoring

### PR DESCRIPTION
**Pull request by SharifChoudhury1**
Automating Claimant Representative and Refactoring

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js [hunk 1]
```diff
@@ -5,13 +5,42 @@ const testConfig = require("../../../config");
...
-    await I.selectOption('#representativeClaimantType_representative_address_representative_address_addressList', commonConfig.claimantRepresentativeAddress);
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.submit)
+    I.selectOption('#representativeClaimantType_representative_address_representative_address_addressList', commonConfig.claimantRepresentativeAddress);
+    I.click(commonConfig.continue);
+
+    I.waitForText('Claimant Representative', testConfig.TestTimeToWaitForText);
+    I.see('Case Number:');
+    I.click(commonConfig.submit);
+    I.see('has been updated with event: Claimant Representative');
```
Comment (pats-john) [inline src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:45]: Can you please Click on the claimant Representative Tab and Check all the relevant Input values are present..?

**Review by pats-john — CHANGES_REQUESTED** [review]

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js (hunk 1 above)
Comment (SharifChoudhury1) [reply src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:45]: Added tab and verified input values

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js [hunk 2]
```diff
@@ -5,13 +5,75 @@ const testConfig = require("../../../config");
...
+
+    I.click("//div[text()='Claimant Representative']");
+    I.see('Claimant Representative Details');
```
Comment (pats-john) [inline src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:48]: WaitForText

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js [hunk 3]
```diff
@@ -5,13 +5,75 @@ const testConfig = require("../../../config");
...
+    I.see('Name of Representative');
+    I.see('Name of Organisation');
+    I.see('Reference');
```
Comment (pats-john) [inline src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:51]: Values Succeed labels....

**Review by pats-john — APPROVED** [review]

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js (hunk 2 above)
Comment (SharifChoudhury1) [reply src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:48]: Added I.waitForText('Claimant Representative Details', testConfig.TestTimeToWaitForText);

File: src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js (hunk 3 above)
Comment (SharifChoudhury1) [reply src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js:51]: Added value and label

**Review by dharrisMOJ — APPROVED** [review]

**Pull request merged**